private-templates-service/client/src/__tests__/App.test.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
```

Comment:
Are you going to have a _tests_ folder that has all the tests, or have each component have it's own test in its own folder?  I commented to Harsh yesterday a pattern I like, but I think it really needs to be defined now before the codebase gets too large 
Relevant discussion:
https://github.com/microsoft/adaptivecards-templates/pull/32#discussion_r372684998

Reply:
I checked in with Harsh and we will place component test's with the component.